### ISSUE TITLE
[qt5-base] build failure

### ISSUE THREAD

**Issue by sweco-nlahoo** [microsoft/vcpkg#14194]
**Host Environment**
- OS: Windows 10
- Compiler: MSVC++ 2019

**To Reproduce**
`./vcpkg install qt5-base:x64-windows`

**Failure logs** 
It's not so much the building but it fails to download some of its dependencies, a broken mirror perhaps?:

```
-- Downloading https://repo.msys2.org/msys/x86_64/pkg-config-0.29.2-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/make-4.3-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libintl-0.19.8.1-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 28;"Timeout was reached"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:181 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues
```

**Additional context**
n/a

**Comment by omarmodaly** [comment]
I had the same issue on **Windows 10** with **MSVC++ 2019**, with the same logs, plus:
```
Error: Building package icu:x64-windows failed with: BUILD_FAILED
```

**Comment by omarmodaly** [comment]
I have the same issue here on **Windows 10** with **MSVC++ 2019**. The logs are exactly the same, plus:
```
Error: Building package icu:x64-windows failed with: BUILD_FAILED
```

**Comment by myearsarealight** [comment]
I'm getting the same thing trying to install icu (x86) and boost-locale[icu], also on Windows 10 and MSVC++ 2019. It happens in Command Prompt too.

**Comment by ddomit** [comment]
I have the same problem

**Comment by JackBoosY** [comment]
Can you manually download the files using the link address?

**Comment by sweco-nlahoo** [comment]
I tried that, but I could not.

Outlook for Android<https://aka.ms/ghei36> downloaden
________________________________

**Comment by turska79** [comment]
I get the same issue when trying to install mlpack. Server has libiconv-1.16-2-x86_64.pkg.tar.zst instead of 1.16-1.

**Comment by ddomit** [comment]
Has someone been able to fix this?

**Comment by JackBoosY** [comment]
Will fix in #14420.
